tests/envs/test_registration.py

```diff
@@ -2,9 +2,9 @@
 
 import gym
 from gym import envs, error
-from gym.envs import registration
+from gym.envs import register, registration, registry, spec
```

Comment:
This shouldn't matter in implementation but in this test you sometimes do `envs.register` and sometimes `register` which got me confused. Could you only one of the method to make it consistent as I was getting confused thinking that they were different

Reply:
Fixed